src/main/java/ua/lviv/iot/parkingServer/model/Record.kt

```diff
@@ -0,0 +1,6 @@
+package ua.lviv.iot.parkingServer.model
+
+abstract class Record {
```

Comment:
`CsvRecord` or `CsvData` would be more clear name

Reply:
✅